README.md

```diff
@@ -63,7 +63,7 @@ host#2$ sudo ip addr add 172.18.200.103/24 dev eth0
 .
 #### Setup the storage disks
 
-The backend containers will need to be provided with the storage (directory) where the data will be persisted. The directory should be accessible via the containers. One host1 and host2 setup creates a directory on the disk that has sufficient space for holding the volume. (Again, these operations are taken care by the Storage Orchestrator - Maya, if you are using Kubernetes).
+The backend containers will need to be provided with the storage (directory) where the data will be persisted. The directory should be accessible via the containers. One host1 and host2 setup create a directory on the disk that has sufficient space for holding the volume. (Again, these operations are taken care by the Storage Orchestrator - Maya, if you are using Kubernetes).
```

Comment:
This doesn't sound good.
Can we change it to , "On host1 and host2 setups create a directory on the disk that has sufficient space for holding the volume."